f8a_worker/models.py

```diff
@@ -138,6 +138,7 @@ class Version(Base):
     id = Column(Integer, primary_key=True)
     package_id = Column(Integer, ForeignKey(Package.id))
     identifier = Column(String(255), index=True)
+    synced2graph = Column(Boolean, nullable=False, default=False)
```

Comment:
@tuxdna 

Question: who is going to set the flag to `False` when we update for example GitHub data for the package that was previously successfully synced to graph?

Reply:
The default value is `False` so existing `INSERT` statements don't need to do anything. The first time an EPV is synced, **data-model-importer** will update the value to `True`. However you are right that, for cases where an existing EPV's data is update by some worker, we need to update this flag back to`False` again.

Reply:
Let me add code set this flag on any update to EPV